scripts/west_commands/runners/spi_burn.py

```diff
@@ -59,10 +62,29 @@ def do_run(self, command, **kwargs):
         else:
             self.logger.error(f'{command} not supported!')
 
+    def start_iceman(self):
+
+        # Start ICEman as background process
+        cmd_ice_run = ["./ICEman", '-Z', 'v5', '-l', 'aice_sdp.cfg']
+        self.ice_process = subprocess.Popen(cmd_ice_run, stdout=subprocess.PIPE)
+
+        # Wait till it ready or exit by timeout
+        start = time.time()
+        while True:
+            out = self.ice_process.stdout.readline()
+            if b'ICEman is ready to use.' in out:
+                break
+            if time.time() - start > self.timeout:
+                raise RuntimeError("TIMEOUT: ICEman is not ready")
+
+    def stop_iceman(self):
+        # Kill ICEman subprocess
+        self.ice_process.terminate()
+
     def _flash(self):
 
-        origin_dir = os.getcwd()
-        iceman_dir = os.path.dirname(self.iceman_path)
+        self.origin_dir = os.getcwd()
+        self.iceman_dir = os.path.dirname(self.iceman_path)
```

Comment:
In the latest round of changes, you moved the only uses of these values back into the `_flash` method. They are not used outside of the method anymore. Therefore, there is no reason to store them as instance attributes anymore either. Please keep these as local variables.

Reply:
Done